src/components/nav-bar/MobileContent.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from '@reach/router';
+import HubblrImage from '../image-components/HubblrImage';
+import HamburgerMenuIcon from '../image-components/HamburgerMenuIcon';
+import AppButton from '../buttons/bases/AppButton';
+
+const MobileContent = ({ isCollapsed, onClick }) => {
```

Comment:
bin immer noch kein Großer Fan vom Naming. `NavBarMobileContent` o.ä., I guess

Reply:
same for desktop

Reply:
Done.